tests/seccomp-profiles/task.yaml

```diff
@@ -0,0 +1,30 @@
+summary: Check seccomp profile generation for apps and hooks.
+prepare: |
+  for snap in basic-binaries basic-hooks
+  do
+    snapbuild ./../fixtures/snaps/$snap .
+  done
+restore: |
+  for snap in basic-binaries basic-hooks
+  do
+    sudo snap remove $snap
+    rm ./${snap}_1.0_all.snap
+  done
+execute: |
+  seccomp_profile_directory="/var/lib/snapd/seccomp/profiles"
+
+  echo "seccomp profiles are generated for apps"
+  sudo snap install ./basic-binaries_1.0_all.snap
+
+  for profile in snap.basic-binaries.block snap.basic-binaries.cat snap.basic-binaries.echo snap.basic-binaries.fail snap.basic-binaries.success
+  do
+    [ -f "$seccomp_profile_directory/$profile" ] || exit 1
```

Comment:
This test is duplicating the whole logic just to add this additional line, it seems?  Can we just add this one line into the other test and rename it to security-profiles?